exercises/practice/two-bucket/src/test/java/TwoBucketTest.java

```diff
@@ -67,6 +73,8 @@ public void testBucketOneSizeOneBucketTwoSizeThreeStartWithTwo() {
 
     @Disabled("Remove to run test")
     @Test
+    @DisplayName("Measure using bucket one of size 2 and bucket two of size 3 - " +
```

Comment:
The message exceeds 120 characters. So either we modify this in the [specification](https://github.com/exercism/problem-specifications/blob/main/exercises/two-bucket/canonical-data.json) or I have to split in two lines